schema.graphql

```diff
@@ -32,3 +35,27 @@ type Account @entity {
   ownedDomains: [Domain!]! @derivedFrom(field: "owner")
   mintedDomains: [Domain!]! @derivedFrom(field: "minter")
 }
+
+type DomainMetadataChanged implements DomainEvent @entity {
```

Comment:
Needs a `metadataUri` string field with the new metadata uri

Reply:
ok, so should I remove the metadata field from the domain entity then or have one in both?

Reply:
Have Both